src/main/java/org/jabref/logic/ai/AiService.java

```diff
@@ -66,17 +67,17 @@ public AiService(AiPreferences aiPreferences, DialogService dialogService, TaskE
 
             mvStore = MVStore.open(mvStorePath.toString());
         } catch (Exception e) {
-            LOGGER.error("An error occurred while creating directories for storing chat history. Chat history won't be remembered in next session", e);
-            dialogService.notify("An error occurred while creating directories for storing chat history. Chat history won't be remembered in next session");
+            LOGGER.error("An error occurred while creating directories for AI cache and chat history. Chat history will not be remembered in next session", e);
+            dialogService.notify(Localization.lang("An error occurred while creating directories for AI cache and chat history. Chat history will not be remembered in next session"));
```

Comment:
Remember that sentences are finished by a dot (`.`).

```suggestion
            dialogService.notify(Localization.lang("An error occurred while creating directories for AI cache and chat history. Chat history will not be remembered in next session."));
```